src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -30,6 +30,16 @@ Copyright (c) .NET Foundation. All rights reserved.
     <StartupHookSupport>false</StartupHookSupport>
   </PropertyGroup>
 
+  <!-- We disable built-in Com support for trimmed apps here so that the feature
+       switch can flow to the runtimeconfig.json. Built-in Com is disabled
+       by default since they may require assemblies, types or members that
+       could be removed by the linker, causing a trimmed app to crash. -->
+  <PropertyGroup Condition="'$(BuiltInComSupport)' == '' And
+                            '$(PublishTrimmed)' == 'true' And
+                            $([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '6.0'))">
```

Comment:
So this property should be set for any framework with version greater than or equal to 6.0? Net framework and net standard as well as net core (in the event those ever exist :) )?

Reply:
Yes, this is the pattern that is followed by similar feature switches like StartupHookSupport that we added recently.